### PR TITLE
Adding iOS support for kotlin multiplatform mobile apps

### PR DESCRIPTION
**Pull request by flixlo**
Hi @murzagalin!

I’m developing a kotlin multiplatform mobile app for Android and iOS and would like to use your library. Unfortunately, the multiplatform-expression-evaluator does not support iOS targets. 

That’s why I created this pull request, which adds the iOS support to the library. I tested the PR with my app and successfully executed the expression parser on both platforms: Android and iOS.

Thanks for your feedback!

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -43,6 +43,13 @@ kotlin {
             }
         }
     }
+    ios {
+        binaries {
+            framework {
+                baseName = "MultiplatformExpressionsEvaluator"
```
Comment (murzagalin) [inline build.gradle.kts]: the name could be just `ExpressionsEvaluator`

**Review by murzagalin — APPROVED** [review]
Thanks, `flixlo` for your contribution!
If you have any other improvements in mind, don't hesitate to create PRs or suggest changes.

File: build.gradle.kts (hunk 1 above)
Comment (flixlo) [reply build.gradle.kts]: Updated as suggested :)

**Comment by murzagalin** [comment]
please use `Squash and merge` option to merge

**Comment by flixlo** [comment]
> please use `Squash and merge` option to merge

I don't think I have the permissions to do that

**Comment by flixlo** [comment]
@murzagalin Thank you very much for the quick response and new release, I appreciate that! :)

**Pull request merged**